app.rb

```diff
@@ -0,0 +1,103 @@
+require './book'
+require './teacher'
+require './rental'
+require './student'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def all_books
+    if @books.empty?
+      puts "There are no books created yet, Add some books to see them here :( \n\n"
+    else
+      @books.each { |bk| puts "\n Title: #{bk.title} by #{bk.author} \n \n" }
+    end
+  end
```

Comment:
- [ ] In the school library console, when 1 is pressed it should be shown all the books from the list. But currently, when I press `"1"` it exits from the console. Kindly fix the issue. :+1: 

![list_all_books](https://user-images.githubusercontent.com/70483835/199808357-54f9749f-f8d5-4f59-9c77-6e8f8d569cb2.png)

Reply:
Noted and fixed. Thanks, for calling my attention to the issue